.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Create a report to help us improve Juju
+labels:
+  - kind/bug
+  - needs-triage
+body:
+  - type: textarea
+    id: Description
+    attributes:
+      label: Description
+      description: "Please describe the bug for us: "
+      placeholder: ex. "`juju status` returns the error `not found`"
+    validations:
+      required: true
+
+  - type: input
+    id: Version
+    attributes:
+      label: Juju version
+      description: "What version of Juju are you using?"
+      placeholder: ex. 3.6.1
+    validations:
+      required: true
+
+  - type: input
+    id: Cloud
+    attributes:
+      label: Cloud
+      description: What cloud are you using?
+      placeholder: ex. AWS
+    validations:
+      required: false
+
+  - type: textarea
+    id: Expected-Behaviour
+    attributes:
+      label: Expected behaviour
+      description: "Describe of what you expected to happen: "
+      placeholder: ex. "`juju status` should tell me what was not found"
+    validations:
+      required: true
+
+  - type: textarea
+    id: Reproduction-Steps
+    attributes:
+      label: "Reproduce / Test"
+      description: "Steps to reproduce the behaviour: "
+      placeholder: ex. "juju bootstrap aws && juju status"
+    validations:
+      required: false
```

Comment:
I do think this part should be mandatory, at least the user should list what *they* did for the bug to appear, if a proper reproduction scenario is too hard.